### DISCUSSION TITLE
Link to official documentation leads to 404 page

### DISCUSSION THREAD

**Discussion by gruberdev** [bluesky-social/atproto#2801] [General Discussion]
[At the Go project's Github page](https://github.com/bluesky-social/indigo/tree/main), there's the following link:
> [atproto.com/docs](https://atproto.com/docs) 
<sub> ([Mirror](https://web.archive.org/web/20240908212115/https://github.com/bluesky-social/indigo/tree/main)) </sub>

[But as you can see, it leads to a 404](https://web.archive.org/web/20240908203345/https://atproto.com/docs).

Albeit a minor detail, I hope this heads up is helpful. Let me know if you have any questions. I didn't find any other thread discussing this as well.

**Comment by bnewbold** [comment]
fixed, thanks!
